demo/index.html

```diff
@@ -4,7 +4,7 @@
 		<meta charset="utf-8">
 		<title>Typography Snapshot</title>
 		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
-		<script src="../../webcomponentsjs/webcomponents-lite.min.js"></script>
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
Totally agree, I've done the same in the component I'm building -- I think you're since the loader only exists in the 1.0 version of webcomponentsjs, not the 0.7 one.